scripts/codegen.sh

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
+
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+set -exEuo pipefail
+
+PROTOC_VERSION='24.3'
```

Comment:
I think we can skip the check on `protoc_version`, it should be the `protoc_version` used by the committer to generate the code, but this is not possible.